nuxeo-admin-console-web/angular-app/src/app/features/home/services/home.service.ts

```diff
@@ -1,18 +1,32 @@
 import { Injectable } from "@angular/core";
 import { HttpClient } from "@angular/common/http";
 import { Observable } from "rxjs";
-import { environment } from "../../../../environments/environment";
+import { map } from "rxjs/operators";
 import { versionInfo } from "../../../shared/types/version-info.interface";
+import { NuxeoJSClientService } from "../../../shared/services/nuxeo-js-client.service";
+import { CapabilitiesResponse } from "../../../shared/types/capabilities.interface";
 
 @Injectable({
   providedIn: "root",
 })
 export class HomeService {
-  private readonly jsonFilePath = environment.apiUrl + "/version-info.json";
+  private readonly capabilities = "capabilities";
 
-  constructor(private http: HttpClient) {}
+  constructor(
+    private http: HttpClient,
+    private nuxeoJsClientService: NuxeoJSClientService
+  ) {}
 
-  getversionInfo(): Observable<versionInfo> {
-    return this.http.get<versionInfo>(this.jsonFilePath);
+  getVersionInfo(): Observable<versionInfo> {
+    return this.http
+      .get<CapabilitiesResponse>(
+        `${this.nuxeoJsClientService.getApiUrl()}${this.capabilities}`
+      )
+      .pipe(
+        map((data) => ({
+          version: data.server.distributionVersion,
+          clusterEnabled: data.cluster.enabled,
+        }))
```

Comment:
Please add ?. for the property values that are assigned to version & clusterEnabled